.github/workflows/scanner-versioned-definitions-update.yaml

```diff
@@ -310,15 +317,14 @@ jobs:
         grep -E "^[0-9]\.[0-9]+\.[0-9]+$" scanner/updater/version/RELEASE_VERSION | while read -r release; do
           case "$release" in
           4.4.*)
+            echo "Copy $release"
             gsutil cp "$single" "gs://definitions.stackrox.io/v4/vulnerability-bundles/$release/"
             ;;
-          4.5*)
+          4.5.*)
```

Comment:
Previously, this would have copied `v1` data into something like `4.50.0`, which should not be touched. We just want to capture `4.5.*` here